pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -415,6 +422,16 @@ func (ctx *context) getDIDDocAndConnection(pubDID string) (*did.Doc, *Connection
 		return nil, nil, fmt.Errorf("create %s did: %w", didMethod, err)
 	}
 
+	// TODO: initialize did map (or mock) in all eleventy billion tests
```

Comment:
?

Reply:
purged this check now.
framework initializes this properly, and we don't put checks in to catch broken tests.